packages/explorer/src/helpers/postMessageRelayHelpers.ts

```diff
@@ -383,7 +385,7 @@ export async function executeOperation({
                 ? {
                     message: err.message,
                     ...('stack' in err && typeof err.stack === 'string'
-                      ? { stack: err.stack }
+                      ? { extensions: { stack: err.stack } }
```

Comment:
This is currently not GraphQL spec compliant, so moving to extensions fixes that